### ISSUE TITLE
Rogue skilltree: Serrated Blades

### ISSUE THREAD

**Issue by Pisshead** [elysium-project/server#2030]
First of all, thank you for taking the time to report this issue.

Please note that the bug tracker does not accept reports on account or character
issues that are unrelated to the server's development. You should open a ticket
in game and wait for the response of a gamemaster. They may redirect you to this
bug tracker if they believe there's a server issue that they cannot help you
with.

Before writing your report, please do read and consider the following points to
increase the quality of your report.

- Search the issue tracker for open and closed issues before opening a new
  issue. You may be able to contribute additional facts by commenting on
  existing issues instead of adding extra noise in a separate issue.

- http://www.wowhead.com/ and http://db.vanillagaming.org/ are not valid sources
  for quests, items etc., instead, please use archived versions of
  http://wow.allakhazam.com or http://www.thottbot.com through
  https://archive.org/.

- Validate the date and time of the archived sources to a certain patch. The
  patch release dates can be found on the corresponding WoWWiki article, as
  found at http://wowwiki.wikia.com/wiki/Category:World_of_Warcraft_patches.

- Videos are valid if and only if the footage contains a timestamp or
  information that confirms the time it was recorded. Date of upload is NOT
  valid and does not confirm when it was shot.

- Read the Effective Bug Reporting article at
  https://github.com/elysium-project/server/blob/development/docs/CONTRIBUTING.md
  which covers these points and more.

Once you've read this message, delete it and write your report below.

--

The tooltip of the skill "Serrated blades" reads "Causes Your attacks to ignore 0 of your target's Armor..." shouldn't "0" be another number? I can't remember which one it would be. I htought it would change as soon as there's a skillpoint set, but it didn't. Does that skill even work?

**Comment by sickheals** [comment]
Do some research and fill in the information that is missing. Re-read the effective bug reporting guide. Post a screenshot to help.

**Comment by GuybrushGit** [comment]
This is a visual bug in the client. Cannot be fixed.

**Comment by luigimannoni** [comment]
I do not agree with the standard answer of "spell tooltips can't be fixed". All spell tooltips were working correctly on official servers. Some mage tooltips as well are broken.

**Comment by Pisshead** [comment]
![grafik](https://user-images.githubusercontent.com/26183948/29994547-40aead86-8fd2-11e7-96db-8eaf341ef81a.png)

**Comment by Pisshead** [comment]
http://wowwiki.wikia.com/wiki/Serrated_Blades
